server/webhook.go

```diff
@@ -118,6 +119,10 @@ func (p *Plugin) handleWebhook(w http.ResponseWriter, r *http.Request) {
 		repoPrivate = event.Project.Visibility == gitlabLib.PrivateVisibility
 		pathWithNamespace = event.Project.PathWithNamespace
 		fromUser = event.User.Username
+		if !p.configuration.EnableChildPipelineNotifications && event.ObjectAttributes.Source == eventSourceParentPipeline {
```

Comment:
@raghavaggarwal2308  Can we add a test case to unit test this?

Reply:
@Kshitij-Katiyar Added unit tests